iris_grib/tests/results/unit/load_cubes/load_cubes/reduced_raw.cml

```diff
@@ -1,33 +1,36 @@
 <?xml version="1.0" ?>
 <cubes xmlns="urn:x-iris:cubeml-0.2">
   <cube standard_name="geopotential" units="m2 s-2">
+    <attributes>
+      <attribute name="centre" value="European Centre for Medium Range Weather Forecasts"/>
+    </attributes>
     <coords>
       <coord>
-        <dimCoord id="1d45e087" points="[0]" shape="(1,)" standard_name="forecast_period" units="Unit('hours')" value_type="int32"/>
+        <dimCoord id="1d45e087" points="[0]" shape="(1,)" standard_name="forecast_period" units="Unit('hours')" value_type="int64"/>
+      </coord>
+      <coord>
+        <dimCoord id="9c8bdf81" points="[377688.0]" shape="(1,)" standard_name="forecast_reference_time" units="Unit('hours since 1970-01-01 00:00:00', calendar='gregorian')" value_type="float64"/>
       </coord>
       <coord datadims="[0]">
-        <auxCoord id="77a50eb5" points="[88.572168514, 88.572168514, 88.572168514, ...,
-		-88.572168514, -88.572168514, -88.572168514]" shape="(13280,)" standard_name="latitude" units="Unit('degrees')" value_type="float64">
+        <auxCoord id="37a3e9a4" points="[88.572168514, 88.572168514, 88.572168514, ...,
+		-88.572168514, -88.572168514, -88.572168514]" shape="(13280,)" standard_name="latitude" units="Unit('degrees_north')" value_type="float64">
```

Comment:
i don't think these should be `units=degrees_north`
where in the code would this behaviour have changed?